extensions/README.md

```diff
@@ -53,6 +53,7 @@ An extension can add new fields to STAC entities (content extension), or can add
 | [EO](eo/README.md) (`eo`)                                    | Item             | Covers electro-optical data that represents a snapshot of the earth for a single date and time. It could consist of multiple spectral bands, for example visible bands, infrared bands, red edge bands and panchromatic bands. The extension provides common fields like bands, cloud cover, off nadir, sun angle + elevation, gsd and more. | *Pilot* |
 | [Point Cloud](pointcloud/README.md) (`pc`)                   | Item             | Provides a way to describe point cloud datasets. The point clouds can come from either active or passive sensors, and data is frequently acquired using tools such as LiDAR or coincidence-matched imagery. | *Proposal* |
 | [SAR](sar/README.md) (`sar`)                                 | Item             | Covers synthetic-aperture radar data that represents a snapshot of the earth for a single date and time. | *Proposal* |
+| [Search](search/README.md) | ItemCollection | Adds search-related metadata to ItemCollection. | *Proposal* |
```

Comment:
What is an ItemCollection? Is this extending the Item or the Collection spec or both?

Reply:
Or is this an API extension? In this case this is not the correct list of extensions...

Reply:
@m-mohr going back to the very first sprint, we originally defined the basic Item structure as a geojson "compatible" structure, where it was not a "Feature" but an "Item".  And all of the items belonged to an "ItemCollection".  However, we wanted to be friendly to clients, so we stuck to using "Feature" and "FeatureCollection" as the type.  If you look at `[/api-spec/STAC.yaml](https://github.com/radiantearth/stac-spec/blob/master/api-spec/STAC.yaml)', you'll see on line 808 that we define an `ItemCollection` with the same definition as a `FeatureCollection`:

```
    itemCollection:
      type: object
      required:
        - features
        - type
      properties:
        type:
          type: string
          enum:
            - FeatureCollection
```

Basically the exact same way we have `items` in our catalogs/collections that all have `"type": "Feature"`.  Kind of an odd nuance, but those are the historical semantics.

Reply:
Fair enough. It's just confusing for new users as this is not defined outside of schemas and there's no ItemCollection spec. I think this should be improved, either reporting it as GeoJSON FeatureCollection or have an ItemCollection spec somewhere and link to it.

Reply:
Totally agree.  It's confusing enough that I'm pretty sure I was the only person to even remember that this is how we defined it.

Reply:
Filed https://github.com/radiantearth/stac-spec/issues/484 -- I'd gladly take this on, but I don't have permissions to assign it to myself.